.github/workflows/functional-tests.yml

```
@@ -29,5 +29,15 @@ jobs:
         find src -name spec.yaml -print0 | xargs -0 -n1 aac validate
         find src/aac/plugins -name *.yaml -print0 | xargs -0 -n1 aac validate
 
+    - name: Validate Example Model
+      run: |
+        find model/flow -name system.yaml -print0 | xargs -0 -n1 aac validate
```

We don't actually need  to use `find` for this since we're targeting one specific file:

```suggestion
        aac validate model/flow/system.yaml
```

---

okay, I see the difference. I was leveraging working code. Thanks for the suggestion.